services/QuillLMS/client/app/bundles/Connect/components/studentLessons/lesson.jsx

```diff
@@ -110,6 +125,62 @@ export class Lesson extends React.Component {
     }
   }
 
+  componentWillUnmount() {
+    window.removeEventListener('keydown', this.resetTimers)
+    window.removeEventListener('mousemove', this.resetTimers)
+    window.removeEventListener('mousedown', this.resetTimers)
+    window.removeEventListener('click', this.resetTimers)
+    window.removeEventListener('keypress', this.resetTimers)
+    window.removeEventListener('scroll', this.resetTimers)
+    window.removeEventListener('visibilitychange', this.setIdle)
```

Comment:
I think we want to constantize these event listener keys in a shared file since we use them in several places